### PR TITLE
hide a TODO comment from the website and review  the command for debugging generated classes

### PR DESCRIPTION
**Pull request by machi1990**
/cc @geoand @gsmet

### CODE REVIEW

File: docs/src/main/asciidoc/extension-authors-guide.adoc [hunk 1]
```diff
@@ -1473,7 +1473,7 @@ memory in a runtime class loader. To have these classes written out to disk for
 
 [source,shell]
 ----
-./mvnw clean install -Dquarkus.debug.generated-classes-dir=./target/app-generated-classes
+mvn clean install -Dquarkus.debug.generated-classes-dir=./target/app-generated-classes
```
Comment (geoand) [inline docs/src/main/asciidoc/extension-authors-guide.adoc]: What's the rationale behind this change?

Comment (machi1990) [reply docs/src/main/asciidoc/extension-authors-guide.adoc]: Hi @geoand thanks for the review. 

This make it possible to C&P the command and launch it from the extension directory where the maven wrapper is not available.

Comment (geoand) [reply docs/src/main/asciidoc/extension-authors-guide.adoc]: Makes sense. I looked at the what is used in other parts of the guide and this is consistent. 👍

**Review by geoand — APPROVED** [review]

**Comment by geoand** [comment]
Thanks!
